src/app/migrations/versions/3cfb8cf58e98_update_session.py

```diff
@@ -0,0 +1,41 @@
+"""Update session
+
+Revision ID: 3cfb8cf58e98
+Revises: 110f14cc6f39
+Create Date: 2021-04-04 01:26:03.892507
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '3cfb8cf58e98'
```

Comment:
if these are constants, should they be in caps? `REVISION` etc

Reply:
also autogenerated by alembic, may be best to not mess with renaming it